rpg_api/tests/pytest/endpoint/base_classes/test_get_base_classes.py

```diff
@@ -0,0 +1,107 @@
+import pytest
+from httpx import AsyncClient
+from fastapi import status
+from rpg_api.db.postgres.factory import factories
+from typing import Any
+
+url = "/api/postgres/base-classes"
+
+
+def get_user_header(token: str) -> dict[str, Any]:
+    """Return access token for given data."""
+    return {"Authorization": f"Bearer {token}"}
+
+
+@pytest.mark.anyio
+async def test_get_base_classes_empty_list(client: AsyncClient) -> None:
+    """Test get all base classes with 0 base classes in the database: 200."""
+
+    response = await client.get(url)
+
+    assert response.status_code == status.HTTP_200_OK
+    assert response.json()["data"] == []
+    assert len(response.json()["data"]) == 0
```

Comment:
use `get_data` from test_utils to make the tests cleaner

```suggestion
@pytest.mark.anyio
async def test_get_base_classes_empty_list(client: AsyncClient) -> None:
    """Test get all base classes with 0 base classes in the database: 200."""

    response = await client.get(url)
    assert response.status_code == status.HTTP_200_OK
    
    response_data = get_data(response)
    assert response_data  == []
```

Reply:
Utilised it on the tests where it was relevant